rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout"],
```

Comment:
Should we use "sibbo-cmp-layout" everywhere instead of ".sibbo-layout"? It seems more reliable to me